packages/preset-mini/src/_variants/pseudo.ts

```diff
@@ -89,6 +87,11 @@ const PseudoClassFunctions = [
   'has',
 ]
 
+const PseudoClassesMulti: Record<string, string[]> = Object.fromEntries([
+  ['selection', ['::selection', ' *::selection']],
+  ['marker', ['::marker', ' *::marker']],
+])
```

Comment:
Because it is different from other pseudos that require `multiPass`, it is classified as a new category.